samples/go/blob-get/blob_get.go

```diff
@@ -51,24 +51,32 @@ func main() {
 	}
 
 	filePath := flag.Arg(1)
+	var file *os.File
+	showProgress := true
 	if filePath == "" {
-		d.CheckErrorNoUsage(errors.New("Empty file path"))
+		file = os.Stdout
+		showProgress = false
+	} else {
+		// Note: overwrites any existing file.
+		var err error
+		file, err = os.OpenFile(filePath, os.O_WRONLY|os.O_CREATE, 0644)
+		d.CheckErrorNoUsage(err)
+		defer file.Close()
 	}
 
-	// Note: overwrites any existing file.
-	file, err := os.OpenFile(filePath, os.O_WRONLY|os.O_CREATE, 0644)
-	d.CheckErrorNoUsage(err)
-	defer file.Close()
-
 	expected := humanize.Bytes(blob.Len())
 	start := time.Now()
 
 	progReader := progressreader.New(blob.Reader(), func(seen uint64) {
```

Comment:
I think it would be better to make `progReader` just `blob.Reader()` in the case that you're writing to stdout.

Like:
```go
file := os.Stdout
blobReader := blob.Reader()

if filePath != "" {
  file := os.OpenFile(...)
  blobReader = progressReader.New(blobReader, func(seen uint64) { ... })
}

io.Copy(file, blobReader)